runtime/runtime.go

```diff
@@ -1061,13 +1062,6 @@ func (r *interpreterRuntime) meteringInterpreterOptions(runtimeInterface Interfa
 			return
 		}
 
-		var err error
-		wrapPanic(func() {
-			err = runtimeInterface.SetComputationUsed(used)
-		})
-		if err != nil {
-			panic(err)
-		}
```

Comment:
Are we sure we want to remove this? The exit handle won't be called when the computation limit exceeded panic below occurs, right?

I guess it's fine, if FVM still performs the logic of `SetComputationUsed` when it encounters a `ComputationLimitExceededError`, but maybe keep the explicit `SetComputationUsed` in both cases (success and limit exceeded)?

Reply:
hmm, how can we change it to make sure it is always called before returning the panic (other errors as well) ?

Reply:
I put it back.